pkg/controllers/csi/provisioner/controller_test.go

```diff
@@ -190,7 +191,7 @@ func TestOneAgentProvisioner_Reconcile(t *testing.T) { //nolint:revive
 				},
 			},
 		)
-		mockClient := &dtclient.MockDynatraceClient{}
+		mockClient := &mockedclient.Client{}
```

Comment:
```suggestion
		mock := &mockedclient.Client{}
```
since it already has name `Client`.

Reply:
Removed the explicit name before, such that the imports name is "mocks" then, but decided for using the "mockedclient" name, as we sometimes import other mock packages and the naming clashes.

But I agree, we could remove the "mockedclient" naming from packages where there are no problems.